packages/astro/src/core/logger/core.ts

```diff
@@ -98,6 +99,27 @@ function padStr(str: string, len: number) {
 	return str + spaces;
 }
 
+export function getEventPrefix({ level, label }: LogMessage) {
```

Comment:
I can't understand what's this function for, especially in the context of a logger, where we don't have any "event" concept exported externally, and it seems to be only an internal thing.

Reply:
This is used by both the console logger and the node logger. The `export` here is so that both loggers can use it, not meant to signal that it's some public API.

I'll add some JSDoc here to the function to clarify.